### DISCUSSION TITLE
custom translation

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#2993] [Q&A]
Hello.

Is it possible to use a custom translation for one or more languages for the frontend of pretix ?
If yes, is it possible so as a future upgrade will not over overlap these customizations ?

Thank you in advance

**Comment by raphaelm** [comment]
Yes, it is possible.
https://docs.pretix.eu/en/latest/admin/config.html#translations

Of course, adjustments might be needed for future updates.

**Reply by alemao8** [reply]
ok I see. Where are the original po files located ?
What do u mean by adjustments for future updates ? Is there an option to bypass the folders with the custom po files or should I take a backup of them before upgrading in order to paste them after the upgrade ?

**Reply by raphaelm** [reply]
The update will not touch your files in that folder, but the update might require new/different strings to be translated 

See also #2946
